/resized_semantic:
  history: keep_last
  depth: 1
  reliability: best_effort
  durability: volatile

/resized_rgb:
  history: keep_last
  depth: 1
  reliability: best_effort
  durability: volatile
